### PR TITLE
CE-3716 Add Community Policy Module

### PR DESCRIPTION
**Pull request by perjg**
New community policy module.

Help page titles for community policy are stored in i18n file.

### CODE REVIEW

File: extensions/wikia/CommunityPage/CommunityPage.i18n.php [hunk 1]
```diff
@@ -67,6 +67,11 @@
 	'communitypage-noofviews' => '$1 views',
 	'communitypage-full-list' => 'View full list',
 	'communitypage-view-more' => 'View more',
+	'communitypage-policy-heading' => 'Community Policy',
+	'communitypage-policy-text' => 'This community has a set of policies that should be followed by all contributors.',
+	'communitypage-policy-edit' => 'Edit',
+	'communitypage-policy-view' => 'View policy »',
+	'communitypage-policy-pagetitle' => 'Community guidelines',
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/CommunityPage.i18n.php]: Can we decide on naming and keep sufixes consistent with other modules? https://github.com/Wikia/app/pull/10522/files
They both seem to have same purpose and format.

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityPolicyModel.class.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		if ( $title instanceof Title ) {
+			return $title->getFullURL();
+		}
+
+		return '';
+	}
+
+	private function getPolicyTextEditLink() {
+		$title = Title::newFromText(
+			'communitypage-policy-text',
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityPolicyModel.class.php]: I think you meant communitypage-policy-pagetitle
If we want to restrict it to MW ns please write so in qqq so translators would know that we don't want article ns prefix as we add it ourselves.

Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityPolicyModel.class.php]: nvm ;p

File: extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityPolicyModel.class.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+			'policyEdit' => wfMessage( 'communitypage-policy-edit' )->plain(),
+			'policyView' => wfMessage( 'communitypage-policy-view' )->plain(),
+			'showEditLink' => in_array( 'sysop', $wgUser->getEffectiveGroups() ),
+		];
+	}
+
+	private function getPolicyLink() {
+		$title = Title::newFromText(
+			wfMessage( 'communitypage-policy-pagetitle' )->inContentLanguage()->plain(),
+			NS_HELP
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityPolicyModel.class.php:21]: If we want to restrict it to Help ns please write so in qqq so translators would know that we don't want article ns prefix as we add it ourselves.

File: extensions/wikia/CommunityPage/styles/components/_community-policy-module.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.CommunityPolicyModule {
+	.community-page-policy-edit {
+		float: right;
+	}
+
+	p.community-page-policy-text {
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/styles/components/_community-policy-module.scss:6]: is p necessary to override?

Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_community-policy-module.scss:6]: unfortunately yes :(

File: extensions/wikia/CommunityPage/templates/contributorsModule.mustache [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<div class="module ContributorsModule">
+<div class="module contributorsModule">
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/templates/contributorsModule.mustache]: https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions
maybe contributors-module?

**Comment by kamilkoterba** [comment]
LGTM

**Comment by kamilkoterba** [comment]
LGTM

**Comment by lukaszkonieczny** [comment]
LGTM

**Comment by kamilkoterba** [comment]
Tested LGTM

**Pull request merged**